contracts/governance/Governor.sol

```diff
@@ -0,0 +1,110 @@
+pragma solidity ^0.5.11;
+pragma experimental ABIEncoderV2;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+/**
+ * @title Governor
+ * @dev The Governor holds the rights to stage and execute contract calls i.e. changing Livepeer protocol parameters. 
+ */
+contract Governor {
+    
+    using SafeMath for uint256;
+
+    address public owner;
+
+    /// @dev mapping of updateHash (keccak256(update) => executeBlock (block.number + delay)
+    mapping(bytes32 => uint256) updates; 
+
+    event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
+
+    event UpdateStaged(Update update, uint256 delay);
+
+    event UpdateExecuted(Update update);
+
+    event UpdateCancelled(Update update);
+
+    struct Update {
+        address[] target;
+        uint256[] value;
+        bytes[] data;
+    }
+
+   /// @notice Throws if called by any account other than the owner.
+    modifier onlyOwner() {
+        require(msg.sender == owner, "unauthorized");
+        _;
+    }
+
+
+    /// @notice Throws if called by any account other than this contract.
+    /// @dev Forces the `stage/execute` path to be used to call functions with this modifier instead of directly.
+    modifier onlyThis() {
+        require(msg.sender == address(this), "unauthorized");
+        _;
+    }
+
+
+    /// @dev The Ownable constructor sets the original `owner` of the contract to the sender account.
+    constructor() public {
+        owner = msg.sender;
+    }
+
+
+    /// @notice Allows the current owner to transfer control of the contract to a newOwner.
+    /// @dev Can only be called through stage/execute, will revert if the caller is not this contract's address.
+    /// @param newOwner The address to transfer ownership to.
+    function transferOwnership(address newOwner) public onlyThis {
+        require(newOwner != address(0), "newOwner is zero");
+        emit OwnershipTransferred(owner, newOwner);
```

Comment:
The convention used in this repo is to emit the event last after any effects & interactions - let's continue to follow that convention here?

Reply:
If that convention requires us to define an intermediate variable `oldOwner` I don't see much purpose in adhering to it since the code in this function is executed atomically anyway.

Reply:
Ah I see. If emitting the event here avoids introducing an additional var then this looks fine to me.